### ISSUE TITLE
TileDownloader not running on T-Deck board ...

### ISSUE THREAD

**Issue by kokoshell** [spleenware/ripple#73]
All compiles and uploads fine in Arduino IDE but nothing comes on serial debug port ...
SD is 32GB  FAT32 fomatted is it ok ?

**Comment by D3VW01F** [comment]
I had to use 16gb or smaller. 32 seems to be too much even though the T-deck can see it. You can try using an SD card partitioner on PC and remove all partitions. Then create new one at 16,384mb or 8,192mb (or smaller) of the unallocated space. Format FAT32, 4096 sector size is friendly, us if possible. 8192 sector size may be forced if you do 16gb partitions (this is built into Windows but some 3rd party software can bypass it). That said, 8192 sector size should still work. 

Once you get the downloader on your Tdeck, put the SD card in, boot it up. Leave it alone and leave it plugged in. Make sure the wifi network info is correct also, and your API key. All you'll see is a light and then black screen, nothing at all, not even a backlight. Wait about 5 or 10 minutes, shut it off. Check the SD card in a PC or phone or tablet, and see if it pulled any tiles. If tiles are there, its working.

NOTE: Its likely only a couple kilobytes after 5 minutes. This is the Achilles heel of this process...It takes about 24-48 hours (ish) to download just 2 gigs. We are pulling files at a snail's pace, less than a kilobite per second, far less, most of the time. Its not the T-Decks fault, or Ripple. This is the limitation of ThunderForest maps service, where the tiles come from. Or well for the free version. It seems paid versions get higher pull speeds and higher tile quotas, but even the cheap membership, which is monthly, is about 125 (USD), or 95 (GBP). And it goes up to 528 USD or 395 GBP. 

In other words, if youre not rich, youre gonna be downloading for days, thats just how its gotta be. Weeks if youre doing large areas of high zooms. Im on day 5. And im just doing one US state at zoom 15 and the USA as a whole in zoom 10. And this is on PC, which appears to be a touch faster. In addition to not being blind (the downloader script for T-deck and other devices has no UI, its just a black screen). And they mention monthly quotas (file download caps?), so dont be surprised if it just stops suddenly, until the month rolls over.
